config.go

```diff
@@ -249,10 +250,10 @@ func parseAndSetDebugLevels(debugLevel string) error {
 // line options.
 //
 // The configuration proceeds as follows:
-//      1) Start with a default config with sane settings
-//      2) Pre-parse the command line to check for an alternative config file
-//      3) Load configuration file overwriting defaults with any specified options
-//      4) Parse CLI options and overwrite/add any specified options
+//  1. Start with a default config with sane settings
+//  2. Pre-parse the command line to check for an alternative config file
+//  3. Load configuration file overwriting defaults with any specified options
+//  4. Parse CLI options and overwrite/add any specified options
```

Comment:
Sorry, my editor is formatting this on its own. Will revert...

Reply:
reverted https://github.com/btcsuite/btcwallet/compare/e1c5660f9c68153d0f426ca9b09c127fbde80fdb..75db9f23dc42c84faa67b7ed9b3367da8f71f94d